lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -12,14 +12,15 @@ const fs = require('fs');
 const glob = require('glob');
 const path = require('path');
 const assert = require('assert');
-const esprima = require('esprima');
+const tsc = require('typescript');
 const collectAndBakeCtcStrings = require('./bake-ctc-to-lhl.js');
 
 const LH_ROOT = path.join(__dirname, '../../../');
-const UISTRINGS_REGEX = /UIStrings = (.|\s)*?\};\n/im;
+const UISTRINGS_REGEX = /UIStrings = .*?\};\n/s;
```

Comment:
as this has evolved we no longer actually use the capture group, the case insensitivity, or the multiline flag. The `s` flag allows `.` to encapsulate `/s` as well.